price_agg.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, render_template
+app = Flask(__name__)
+
+@app.route("/")
+def home():
+    return render_template('home.html')
+
+if __name__ == '__main__':
+    app.run(debug=True)
```

Comment:
See that this is just a filename change. I don't care what the filename is, but the newline comment remains the same.